NuGet.Config

```diff
@@ -2,8 +2,7 @@
 <configuration>
   <packageSources>
     <clear/>
-    <add key="CI Builds (aspnetvnext)" value="https://www.myget.org/F/aspnetvnext/api/v3/index.json" />
-    <add key="CI Builds (aspnetcidev)" value="http://myget.org/F/aspnetcidev/api/v3/index.json" />
+    <add key="CI Builds (aspnetvnext)" value="https://www.myget.org/F/aspnetrelease/api/v3/index.json" />
```

Comment:
Sure. I could change the dependency to -rc2-* to ensure it doesn't pick up newer packages.